db/migrate/20230718082519_add_primary_key_to_backend_events.rb

```diff
@@ -0,0 +1,25 @@
+class AddPrimaryKeyToBackendEvents < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
OK, it works with `SAFETY_ASSURED=1`, just the error message is misleading

Reply:
Yes, it's the `strong_migrations` gem alerting of a potentially dangerous operation. You can bypass the control by running:

`SAFETY_ASSURED=1 bundle exec rails db:migrate`

However, it already removed the index so it will fail anyway if you try to run migration again. So you need to manually add the index before trying. Or resetting the DB to before the migration.

That's why I was complaining about not having DDL transactions.

Reply:
Yep, I see :upside_down_face: